unstable/wlr-export-dmabuf-unstable-v1.xml

```diff
@@ -59,7 +59,7 @@
     </request>
   </interface>
 
-  <interface name="zwlr_export_dmabuf_frame_v1" version="1">
+  <interface name="zwlr_export_dmabuf_frame_v1" version="2">
```

Comment:
The minor version of the `zwlr_export_dmabuf_manager_v1` interface needs to be bumped as well (since versions are inherited from the parent object).